benchmarks/data_ingestion/benchmark_microgrid_data.py

```diff
@@ -190,14 +190,15 @@ def __init__(self) -> None:
     def component_data_peekable(self) -> None:
         """Mock component_data_peekable method on MicrogridApiClient."""
 
-    async def components(self) -> Set[Component]:
+    async def components(self, timeout: float = 60.0) -> Set[Component]:
```

Comment:
Maybe put the `60.0` in a *constant* here too?

Reply:
Done